File: Beecrowd/1049 Animal/main.c
#include <stdio.h>
#include <string.h>

int main() {

    char A[20], B[20], C[20];

    fflush(stdin);
    gets(A);

    fflush(stdin);
    gets(B);

    fflush(stdin);
    gets(C);

    if (strcmp(A, "vertebrado") == 0 && strcmp(B, "ave") == 0 && strcmp(C, "carnivoro") == 0)
        printf("aguia\n");
    else if (strcmp(A, "vertebrado") == 0 && strcmp(B, "ave") == 0 && strcmp(C, "onivoro") == 0)
        printf("pomba\n");
    else if (strcmp(A, "vertebrado") == 0 && strcmp(B, "mamifero") == 0 && strcmp(C, "onivoro") == 0)
        printf("homem\n");
    else if (strcmp(A, "vertebrado") == 0 && strcmp(B, "mamifero") == 0 && strcmp(C, "herbivoro") == 0)
        printf("vaca\n");
    else if (strcmp(A, "invertebrado") == 0 && strcmp(B, "inseto") == 0 && strcmp(C, "hematofago") == 0)
        printf("pulga\n");
    else if (strcmp(A, "invertebrado") == 0 && strcmp(B, "inseto") == 0 && strcmp(C, "herbivoro") == 0)
        printf("lagarta\n");
    else if (strcmp(A, "invertebrado") == 0 && strcmp(B, "anelideo") == 0 && strcmp(C, "hematofago") == 0)
        printf("sanguessuga\n");
    else if (strcmp(A, "invertebrado") == 0 && strcmp(B, "anelideo") == 0 && strcmp(C, "onivoro") == 0)
        printf("minhoca\n");

    return 0;
}

File: Beecrowd/1049 Animal/Main.java
import java.io.IOException;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);

        String a, b, c;

        a = sc.next();
        b = sc.next();
        c = sc.next();

        if (a.equals("vertebrado") && b.equals("ave") && c.equals("carnivoro"))
            System.out.println("aguia");
        else if (a.equals("vertebrado") && b.equals("ave") && c.equals("onivoro"))
            System.out.println("pomba");
        else if (a.equals("vertebrado") && b.equals("mamifero") && c.equals("onivoro"))
            System.out.println("homem");
        else if (a.equals("vertebrado") && b.equals("mamifero") && c.equals("herbivoro"))
            System.out.println("vaca");
        else if (a.equals("invertebrado") && b.equals("inseto") && c.equals("hematofago"))
            System.out.println("pulga");
        else if (a.equals("invertebrado") && b.equals("inseto") && c.equals("herbivoro"))
            System.out.println("lagarta");
        else if (a.equals("invertebrado") && b.equals("anelideo") && c.equals("hematofago"))
            System.out.println("sanguessuga");
        else if (a.equals("invertebrado") && b.equals("anelideo") && c.equals("onivoro"))
            System.out.println("minhoca");

        sc.close();
    }

}